src/data.py

```diff
@@ -67,7 +67,8 @@ def merge_hours(eateries):
       base_event = operating_hour.events[0]
       for event in operating_hour.events[1:]:  # iterate over copy of list so we can safely remove
         if event.start_time == base_event.end_time and \
-            (not event.menu or event.menu[0].equals(base_event.menu[0])):
+            (not event.menu or (len(event.menu) > 0 and len(base_event.menu) > 0
+                and event.menu[0].equals(base_event.menu[0]))):
```

Comment:
why not:

```python
if (event.start_time == base_event.end_time and 
    event.menu and 
    base_event.menu and 
    event.menu[0].equals(base_event.menu[0])):
...
```

Doesn't this accomplish the same thing while being easier to read?